lib/standard/string.nit

```diff
@@ -1163,10 +1163,13 @@ class FlatString
 	#              String Specific Methods           #
 	##################################################
 
-	private init with_infos(items: NativeString, len: Int, from: Int, to: Int)
+	# Create a new string with given data.
+	#
+	# `items` will be used as is to retrieve the characters of the string.
```

Comment:
This sentence scares me.

I don't know many methods that do not use args **as is**. So, or this sentence means nothing, or something really complex happens and I don't understand.